### ISSUE TITLE
Draconic armor don't remove mining slowdown

### ISSUE THREAD

**Issue by ArthixFR** [gildedgames/aether-issues#621]
Hi,

Arfter somes test with DraconicEvolution and AetherII mods, I found a little bug with draconic armors. Normally when you have this armor you don't have any mining slowdown when you fly or you are underwater. I tried to remove the aether mod and it works.

**Comment by sliderhostmimic** [comment]
This isn't relevant to the Aether II mod.

**Comment by Collin1971** [comment]
@sliderhostmimic: This issue _is_ relevant to the Aether.

**Comment by ArthixFR** [comment]
Yeah I can confirm, I opened an issue on the side of DraconicEvolution, this is not his fault.
